data/io.elementary.calendar.appdata.xml.in

```diff
@@ -171,11 +171,11 @@
     <content_attribute id="money-purchasing">none</content_attribute>
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
-  <developer_name>elementary LLC.</developer_name>
-  <url type="homepage">http://elementary.io/</url>
+  <developer_name>elementary, Inc.</developer_name>
+  <url type="homepage">https://elementary.io/</url>
   <url type="bugtracker">https://github.com/elementary/calendar/issues</url>
   <url type="donation">https://elementary.io/get-involved#funding</url>
-  <url type="help">https://elementary.io/help/maya</url>
```

Comment:
I think the right approach here is to update the website redirect—isn't that why we used our own domain here in the first place?

Reply:
Maybe? But as far as I can tell most appdata doesn't include anything like this